cmux-tui/spec/cli.md

```
@@ -93,8 +94,8 @@ The generated CLI requires one of `--index` or `--delta` for `select-tab`, `sele
 | `move-tab` | implemented | `--surface <id> --pane <id> --index <n>` | none | none |
 | `move-workspace` | implemented | `--workspace <id> --index <n>` | none | none |
 | `scroll-surface` | implemented | `--surface <id> --delta <n>` | none | none |
-| `subscribe` | implemented | none | none in v5 | event JSON lines |
-| `attach-surface` | implemented | `--surface <id>` | none | event JSON lines |
+| `subscribe` | implemented; tree deltas protocol 7 | none | `--tree-events coarse|deltas` | event JSON lines |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)